src/config/data.js

```diff
@@ -125,8 +118,9 @@ export const trainings = [
     location: 'Makers Academy',
     pathUrl: '/react-redux-training-london',
     trainingInstanceId: '5be3366f1307cabce7761593',
-    price: 829.17,
-    ticketName: 'Standard ticket',
+    price: 1166.67,
+    discountPrice: 829.17,
+    priceGoesUpOn: new Date('Dec 10, 2018'),
```

Comment:
@ric9176 this course has 3 prices: 829.17, 1000, 1166.67. The course starts on Jan 22. Should we increase the price a bit later? Dec 17? and then the final price around Jan 12?

Reply:
:100: